distributions/demo-server/src/main/resources/config/default_retry.yaml

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Maybe we can add documentation here on format.

Reply:
YAML formatting is pretty standard, isn't it?